builders/tools/sidecar.md

```diff
@@ -143,36 +200,98 @@ To obtain the EVM sender address, recipient address, and EVM hash, check the `ev
+    | `GasLimit`             | extrinsics.{extrinsic number}.args.transaction.eip1559.gasLimit|
+    | `Access List`          | extrinsics.{extrinsic number}.args.transaction.eip1559.accessList|       
+    | `Signature`            | extrinsics.{extrinsic number}.args.transaction.eip1559.oddYParity/r/s|
+    | `Sender Address`       | extrinsics.{extrinsic number}.events.{event number}.data.0 |
+    | `Recipient Address`    | extrinsics.{extrinsic number}.events.{event number}.data.1 |
+    | `EVM Hash`             | extrinsics.{extrinsic number}.events.{event number}.data.2 |
+    | `EVM Execution Status` | extrinsics.{extrinsic number}.events.{event number}.data.3 |
+
+=== "Legacy"
+    | EVM Field              | Block JSON Field                                           |
+    |------------------------|------------------------------------------------------------|
+    | `Nonce`                | extrinsics.{extrinsic number}.args.transaction.legacy.nonce|
+    | `GasPrice`             | extrinsics.{extrinsic number}.args.transaction.legacy.gasPrice|
+    | `GasLimit`             | extrinsics.{extrinsic number}.args.transaction.legacy.gasLimit|
+    | `Value`                | extrinsics.{extrinsic number}.args.transaction.legacy.value|   
+    | `Signature`            | extrinsics.{extrinsic number}.args.transaction.legacy.signature|
+    | `Sender Address`       | extrinsics.{extrinsic number}.events.{event number}.data.0 |
+    | `Recipient Address`    | extrinsics.{extrinsic number}.events.{event number}.data.1 |
+    | `EVM Hash`             | extrinsics.{extrinsic number}.events.{event number}.data.2 |
+    | `EVM Execution Status` | extrinsics.{extrinsic number}.events.{event number}.data.3 |
+
+=== "EIP2930"
+    | EVM Field              | Block JSON Field                                           |
+    |------------------------|------------------------------------------------------------|
+    | `Chain ID`             | extrinsics.{extrinsic number}.args.transaction.eip2930.chainId|
+    | `Nonce`                | extrinsics.{extrinsic number}.args.transaction.eip2930.nonce |
+    | `GasPrice`             | extrinsics.{extrinsic number}.args.transaction.eip2930.gasPrice|
+    | `GasLimit`             | extrinsics.{extrinsic number}.args.transaction.eip2930.gasLimit|
+    | `Value`                | extrinsics.{extrinsic number}.args.transaction.eip2930.value| 
+    | `Access List`          | extrinsics.{extrinsic number}.args.transaction.eip2930.accessList|       
+    | `Signature`            | extrinsics.{extrinsic number}.args.transaction.eip2930.oddYParity/r/s|
+    | `Sender Address`       | extrinsics.{extrinsic number}.events.{event number}.data.0 |
+    | `Recipient Address`    | extrinsics.{extrinsic number}.events.{event number}.data.1 |
+    | `EVM Hash`             | extrinsics.{extrinsic number}.events.{event number}.data.2 |
+    | `EVM Execution Status` | extrinsics.{extrinsic number}.events.{event number}.data.3 |
```

Comment:
just wanted to double check on this, some of these variables in the "EVM Field" column have spaces in them others don't

Reply:
So the left column is just what these fields represent, and the right is what they are mapped to in the actual JSOn object. I can put a space between fields like GasPrice or GasLimit to be more clear.